builtin/providers/azurerm/resource_arm_network_interface_card.go

```diff
@@ -373,9 +396,10 @@ func validateNetworkInterfacePrivateIpAddressAllocation(v interface{}, k string)
 	return
 }
 
-func expandAzureRmNetworkInterfaceIpConfigurations(d *schema.ResourceData) ([]network.InterfaceIPConfiguration, error) {
+func expandAzureRmNetworkInterfaceIpConfigurations(d *schema.ResourceData) ([]network.InterfaceIPConfiguration, *[]string, error) {
```

Comment:
Why return a pointer to a slice? A slice is basically a pointer already, so I don't really see the benefit.